redwood-layout/README.md

```diff
@@ -0,0 +1,102 @@
+Redwood Layout
+=======
+
+This artifact includes `Row` and `Column` widget implementations for Android Views, Android Compose UI, and iOS UIKit.
+
+To being, add the follow entries to your `schema.kt`:
+
+```kotlin
+@Widget(1)
+data class Row(
+  @Property(1) @Default("Padding.Zero") val padding: Padding,
+  @Property(2) @Default("Overflow.Clip") val overflow: Overflow,
+  @Property(3) @Default("MainAxisAlignment.Start") val horizontalAlignment: MainAxisAlignment,
+  @Property(4) @Default("CrossAxisAlignment.Start") val verticalAlignment: CrossAxisAlignment,
+  @Children(5) val children: RowScope.() -> Unit,
+)
+
+object RowScope {
+  /** https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow */
+  fun LayoutModifier.grow(value: Int): LayoutModifier {
+    return then(GrowLayoutModifier(value))
+  }
+
+  /** https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink */
+  fun LayoutModifier.shrink(value: Int): LayoutModifier {
+    return then(ShrinkLayoutModifier(value))
+  }
+
+  /** https://developer.mozilla.org/en-US/docs/Web/CSS/align-self */
+  fun LayoutModifier.verticalAlignment(alignment: CrossAxisAlignment): LayoutModifier {
+    return then(VerticalAlignmentLayoutModifier(alignment))
+  }
+}
+
+@Widget(2)
+data class Column(
+  @Property(1) @Default("Padding.Zero") val padding: Padding,
+  @Property(2) @Default("Overflow.Clip") val overflow: Overflow,
+  @Property(3) @Default("CrossAxisAlignment.Start") val horizontalAlignment: CrossAxisAlignment,
+  @Property(4) @Default("MainAxisAlignment.Start") val verticalAlignment: MainAxisAlignment,
+  @Children(5) val children: ColumnScope.() -> Unit,
+)
+
+object ColumnScope {
+  /** https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow */
+  fun LayoutModifier.grow(value: Int): LayoutModifier {
+    return then(GrowLayoutModifier(value))
+  }
+
+  /** https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink */
+  fun LayoutModifier.shrink(value: Int): LayoutModifier {
+    return then(ShrinkLayoutModifier(value))
+  }
+
+  /** https://developer.mozilla.org/en-US/docs/Web/CSS/align-self */
+  fun LayoutModifier.horizontalAlignment(alignment: CrossAxisAlignment): LayoutModifier {
+    return then(HorizontalAlignmentLayoutModifier(alignment))
+  }
+}
+```
+
+Then copy the following classes into the `commonMain` of your `widget` module. These classes connect
+the generated schema with the platform `RowWidget` and `ColumnWidget` implementations:
+
+```kotlin
+class RowBridge<T : Any>(private val delegate: RowWidget<T>) : Row<T>, Widget<T> by delegate {
```

Comment:
Was hoping to avoid copying and pasting these classes, but the implementation classes can't implement an interface generated by the schema. Could potentially make the implementations open and mix in the generated interface (i.e. `fun Column() = object : ComposeColumn(), Column<() -> Unit> {}`), but I'd rather not make the implementations `open`.